### PR TITLE
Add IHME Explorer visualization

### PR DESCRIPTION
**Pull request by lossyrob**
This PR adds a new map that displays IHME projections through time. It includes options to choose between metrics, bounds (lower, mean, upper), national or regional aggregations, and displaying data as per-capita or total numbers.

The population data is taken from US census data for states, and via a WorldPop calculation done by @echeipesh via the [geotrellis-worldpop](https://github.com/echeipesh/geotrellis-worldpop) project.

The per-capita base is configured in the Process IHME Data notebook, and is set to "per 100,000" for regions and "per 1,000,000" for countries.

Vector tiles are baked form national and regional GeoJSONS produced by the notebook. Each feature in those datasets contains the location, population, and series of metrics with the property key format as `{date}_{metric}_{boundLevel}`. Vector Tiles do not support nested GeoJSON properties so this property structure lets us utilize the vector tiles to do property filtering based on the date, metric and bound level selected in the UI.

Color breaks are computed using Natural Jenks breaks via [jenkspy](https://github.com/mthh/jenkspy). They are computed for per-capita and totals for regions and nations, and placed into a configuration file that is read in by the front end, which performs the per-capita calculation on the fly based on the metric value, population, and per capita base for each feature.

These vector tiles are heavy and so the time sliding might be slow on machines with less CPU power. I implemented a timer mechanism on the slider so that the slider can only change the date value every 100 ms in an attempt to help with rendering; further optimizations can be made to help with the display speed.

Other future work includes generating a region detail view that provides and Overlay on the map containing charts over time for that region's IHME prediction output.

### CODE REVIEW

**Comment by jfrankl** [comment]
Hey @lossyrob, this is cool. I want to give some quick design feedback. Feel free to take it leave it. If you are interested in a bigger UX review, let me know and I'd be happy to share additional thoughts.

Something weird is happening for me when I use the date picker.
image

I don't see anything interesting on the map before March and after July. If that's always the case, consider removing the other dates from the slider/picker.

Remove the link to http://www.healthdata.org/covid from the heading and put it on some of the body text. It's uncommon to have a link in a heading.

**Comment by lossyrob** [comment]
@jfrankl I addressed 2 of the 3 comments. For the date range one - those dates come from the model, and I think for representation's sake I'd want to keep the model output dates as the full range and not make editorial decisions around that. 

Thank you!

File: maps/_ihme-explorer/src/utils/config.js [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+        label: 'National'
+    },
+    'region': {
+        label: 'Regional',
+        default: true
+    }
+};
+
+// Keys match IHME columns
+export const indicators = {
```
Comment (daveluo) [inline maps/_ihme-explorer/src/utils/config.js:40]: I think we should visually reorder these indicators to match up more closely with what they show on https://covid19.healthdata.org/ vs what's given in their data csv/info sheet.

Suggested reordering:
`deaths` <--also suggest making this the indicator to show by default
`admis`
`newICU`
`allbed`
`ICUbed`
`InvVen`
`bedover`
`icuover`
`totdea`

File: maps/_ihme-explorer/src/utils/config.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+    'allbed': {
+        label: 'All Beds',
+        colors: ['#fff7fb', '#9db5ce', '#4d7596', '#023858'],
+        description: 'Hospital beds needed per day for COVID-19 patients'
+    },
+    'ICUbed': {
+        label: 'ICU Beds',
+        colors: ['#f7fcfd', '#b0aacb', '#7a5a8d', '#4d004b'],
+        description: 'ICU beds needed per day for COVID-19 patients',
+        default: true
```
Comment (daveluo) [inline maps/_ihme-explorer/src/utils/config.js:50]:
```suggestion
        description: 'ICU beds needed per day for COVID-19 patients'
```

**Pull request merged**